README.md

```diff
@@ -0,0 +1,37 @@
+# ElasticSearch on Rancher
+
+> Rancher configuration for running a Elasticsearch cluster
+
+## Services deployed:
+
+- `es-master`, Elasticsearch nodes, part of the Elasticsearch cluster, one runs on every host that has an `escluster=true` tag defined.
+- Justwatch [Exporter], exposing Prometheus metrics on the cluster status.
+- es-client, Rancher cattle DNS service, which points the `es-client` domain name to `es-master`, for backwards compatibility with existing services.
+
```

Comment:
Would prefer a colon or a hyphen after the first word,  as it reads a little strange with a comma. something like
`es-master`: Elasticsearch nodes. Part of the Elasticsearch cluster, one runs on every host that has an `escluster=true` tag defined.